ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.sass

```diff
@@ -16,17 +16,21 @@
   text-align: center
   margin-bottom: 0
   font-size: $title
+.filter-sort
+  height: 5rem
 .form-row
   display: flex
   justify-content: center
   align-items: center
+  @include flexbox(space-between)
   .icon-size
     width: 5rem
 .scroll-marker
   min-height: $height-empty-block
 .trips
-  height: calc(#{$trip-card-height}*5 + #{$height-empty-block})
+  height: calc(#{$trip-card-height}*4 + #{$height-empty-block})
```

Comment:
pls create sass constant for 4 - $number-of-trips-in-list and use it here + remove comment

Reply:
fixed